### ISSUE TITLE
razor pages partial loading via ajax

### ISSUE THREAD

**Issue by adamfoneil** [RyanTaite/DynamicListExample#1]
Hi there, I saw you entered this issue
https://github.com/aspnet/AspNetCore.Docs/issues/8132

Did you ever get anywhere with this? I've been through a ton of runaround on this topic

**Comment by RyanTaite** [comment]
It's been a while since I've looked into this problem. Have you given this project a try to see if it solves your problem? Otherwise, if you give me some details to what you're trying to achieve, I'll see if I can help later when I'm available.

**Comment by adamfoneil** [comment]
Thank you, no I didn't look too hard at your project since I was mid-rampage on my problem. :-) I eventually gave up on the Razor Pages approach and did what I knew how with old-school controller and view. I did go back and look at your project, but had some dependency weirdness -- packages that wouldn't restore or something

![image](https://user-images.githubusercontent.com/4549398/56243270-2da27e80-6068-11e9-86b3-092790286c90.png)

Thanks for replying!
